### PR TITLE
[german translations] some suggestions

### PR DESCRIPTION
**Pull request by Neumann-A**
Note: Some English msgs need to be corrected. 

- Tripel -> Triplett. 
- Need some German definition of `Ports`. Packages/Ports are inconsistently used, maybe just use packages always?
- inconsistency about `Toolsetversion - Toolkettenkombination`.  Toolkette is Denglish. 
- some sentences should be simplified. 

(this PR probably needs to be applied somewhere else.)

And let me CC some germans:
@autoantwort @dg0yt

### CODE REVIEW

File: locales/messages.de.json [hunk 1]
```diff
@@ -1,35 +1,35 @@
 {
   "AddPortSucceded": "Der Datei „vcpkg.json“ konnten erfolgreich Ports hinzugefügt werden.",
-  "AddTripletExpressionNotAllowed": "Fehler: Tripelausdrücke sind hier nicht zulässig. Erwägen Sie, „{package_name}:{triplet}“ in „{package_name}“ zu ändern.",
+  "AddTripletExpressionNotAllowed": "Fehler: Triplettausdrücke sind hier nicht zulässig. Erwägen Sie, „{package_name}:{triplet}“ in „{package_name}“ zu ändern.",
   "AllFormatArgsRawArgument": "Formatzeichenfolge „{value}“ enthält ein nicht formatiertes Formatargument.",
   "AllFormatArgsUnbalancedBraces": "nicht ausgeglichene Klammer in der Formatzeichenfolge „{value}“",
   "AwsAttemptingToFetchPackages": "Es wird versucht, {count} Pakete aus AWS abzurufen.",
-  "AwsFailedToDownload": "AWS konnte nicht herunterladen, Exitcode: {exit_code}",
-  "AwsRestoredPackages": "{count} Pakete von AWS-Servern wurden in {elapsed} wiederhergestellt.",
-  "AwsUploadedPackages": "Binärdateien wurden auf {count} AWS-Server hochgeladen",
```
Comment (Neumann-A) [inline locales/messages.de.json:9]: This is/was logically wrong.

File: locales/messages.de.json [hunk 2]
```diff
@@ -51,23 +51,23 @@
...
+  "UnsupportedToolchain": "Fehler: Im Triplett {triplet}: Es wurde keine gültige Toolset-Kombination gefunden.\n    Die angeforderte Zielarchitektur war {arch}\n    Die ausgewählte Visual Studio-Instanz befindet sich unter {path}\n    Die verfügbaren Toolset-Kombinationen sind {list}\n",
   "UsingManifestAt": "Die Manifestdatei unter {path} wird verwendet.",
   "VSExaminedInstances": "Die folgenden Visual Studio-Instanzen wurden berücksichtigt:",
   "VSExaminedPaths": "Die folgenden Pfade wurden für Visual Studio-Instanzen untersucht:",
   "VSNoInstances": "Es konnte keine vollständige Visual Studio-Instanz gefunden werden.",
-  "VcpkgDisallowedClassicMode": "Fehler: Es wurde kein Manifest (vcpkg.json) oberhalb des aktuellen Arbeitsverzeichnisses gefunden.\nDiese vcpkg-Verteilung verfügt nicht über eine Instanz im klassischen Modus.",
+  "VcpkgDisallowedClassicMode": "Fehler: Es wurde kein Manifest (vcpkg.json) oberhalb des aktuellen Arbeitsverzeichnisses gefunden.\nDiese vcpkg-Verteilung(ROFL) verfügt nicht über eine Instanz im klassischen Modus.",
```
Comment (Neumann-A) [inline locales/messages.de.json:65]: This needs a different wording. vcpkg is probably not a Distribution in the sense of a Gaussian distribution

File: locales/messages.de.json [hunk 3]
```diff
@@ -51,23 +51,23 @@
   "LicenseExpressionUnknownException": "Unbekannter Lizenzausnahmebezeichner „{value}“. Bekannte Werte werden unter https://spdx.org/licenses/exceptions-index.html aufgeführt.",
   "LicenseExpressionUnknownLicense": "Unbekannter Lizenzbezeichner „{value}“. Bekannte Werte werden unter https://spdx.org/licenses/ aufgeführt.",
   "NoLocalizationForMessages": "Keine Lokalisierung für die folgenden Nachrichten:",
-  "ProcessorArchitectureMalformed": "Fehler beim Analysieren von %PROCESSOR_ARCHITECTURE% ({arch}) als eine gültige CPU-Architektur.",
+  "ProcessorArchitectureMalformed": "Fehler beim parsen von %PROCESSOR_ARCHITECTURE% ({arch}) als eine gültige CPU-Architektur.",
```
Comment (Neumann-A) [inline locales/messages.de.json:54]: according to the Duden `parsen` is a German word.

File: locales/messages.de.json [hunk 4]
```diff
@@ -1,35 +1,35 @@
...
+  "ErrorVcvarsUnsupported": "Fehler: im Triplett {triplet}: Die Verwendung der Entwickleraufforderung von Visual Studio wird auf Nicht-Windows-Hosts nicht unterstützt.\nDefinieren Sie „VCPKG_CMAKE_SYSTEM_NAME“ oder „VCPKG_CHAINLOAD_TOOLCHAIN_FILE“ in der Tripelettdatei.",
   "ForceSystemBinariesOnWeirdPlatforms": "Die Umgebungsvariable VCPKG_FORCE_SYSTEM_BINARIES muss auf ARM-, S390x- und ppc64le-Plattformen festgelegt werden.",
   "FormattedParseMessageExpression": "    beim Ausdruck: {value}",
   "FormattedParseMessageLocation": "{path}:{row}:{column}: ",
   "GenerateMsgErrorParsingFormatArgs": "Fehler: Formatzeichenfolge für {value} wird analysiert:",
   "GenerateMsgIncorrectComment": "Die Nachricht {value} weist einen falschen Kommentar auf:",
   "GenerateMsgNoArgumentValue": "    {{{value}}} wurde in einem Kommentar angegeben, aber nicht in der Nachricht verwendet.",
   "GenerateMsgNoCommentValue": "    {{{value}}} wurde in der Nachricht verwendet, aber nicht kommentiert.",
-  "GraphCycleDetected": "Zyklus innerhalb des Diagramms erkannt bei {package_name}:",
```
Comment (Neumann-A) [inline locales/messages.de.json:32]: No it is not a diagram. It is a graph.

File: locales/messages.de.json (hunk 3 above)
Comment (dg0yt) [reply locales/messages.de.json:54]: But here used as a noun, so please Upper Case: `Parsen`

File: locales/messages.de.json [hunk 5]
```diff
@@ -51,23 +51,23 @@
   "LicenseExpressionUnknownException": "Unbekannter Lizenzausnahmebezeichner „{value}“. Bekannte Werte werden unter https://spdx.org/licenses/exceptions-index.html aufgeführt.",
   "LicenseExpressionUnknownLicense": "Unbekannter Lizenzbezeichner „{value}“. Bekannte Werte werden unter https://spdx.org/licenses/ aufgeführt.",
   "NoLocalizationForMessages": "Keine Lokalisierung für die folgenden Nachrichten:",
-  "ProcessorArchitectureMalformed": "Fehler beim Analysieren von %PROCESSOR_ARCHITECTURE% ({arch}) als eine gültige CPU-Architektur.",
+  "ProcessorArchitectureMalformed": "Fehler beim parsen von %PROCESSOR_ARCHITECTURE% ({arch}) als eine gültige CPU-Architektur.",
   "ProcessorArchitectureMissing": "Die erforderliche Umgebungsvariable %PROCESSOR_ARCHITECTURE% fehlt.",
-  "ProcessorArchitectureW6432Malformed": "Fehler beim Analysieren von %PROCESSOR_ARCHITEW6432% ({arch}) als eine gültige CPU-Architektur. Fallback auf %PROCESSOR_ARCHITECTURE% wird durchgeführt.",
+  "ProcessorArchitectureW6432Malformed": "Fehler beim parsen von %PROCESSOR_ARCHITEW6432% ({arch}) als eine gültige CPU-Architektur. Fallback auf %PROCESSOR_ARCHITECTURE% wird durchgeführt.",
```
Comment (dg0yt) [inline locales/messages.de.json:56]: Upper Case: `Parsen`

File: locales/messages.de.json (hunk 2 above)
Comment (dg0yt) [reply locales/messages.de.json:65]: Maybe `vcpkg-Installation`

File: locales/messages.de.json [hunk 6]
```diff
@@ -51,23 +51,23 @@
...
+  "VcpkgDisallowedClassicMode": "Fehler: Es wurde kein Manifest (vcpkg.json) oberhalb des aktuellen Arbeitsverzeichnisses gefunden.\nDiese vcpkg-Verteilung(ROFL) verfügt nicht über eine Instanz im klassischen Modus.",
   "VcpkgHasCrashed": "vcpkg.exe ist abgestürzt.\nBitte senden Sie eine E-Mail an:\n    {email}\nmit einer kurzen Zusammenfassung dessen, was Sie versucht haben, und dem folgende Datenblob:",
   "VcpkgHasCrashedArgument": "{value}|",
   "VcpkgHasCrashedDataBlob": "\nVersion={version}\nEXCEPTION='{error}'\nCMD=",
   "VcpkgInvalidCommand": "Ungültiger Befehl: {command_name}",
   "VcpkgSendMetricsButDisabled": "Warnung: --sendmetrics wurde übergeben, Metriken sind jedoch deaktiviert.",
-  "VersionSpecMismatch": "Fehler: Fehler beim Laden des Ports, weil die Versionsspezifikationen nicht übereinstimmen\n    Pfad: {path}\n    Erwartet: {expected_version}\n    Tatsächlich: {actual_version}",
+  "VersionSpecMismatch": "Fehler: Ports konnten nicht geladen werden, weil die Versionsspezifikationen nicht übereinstimmen\n    Pfad: {path}\n    Erwartet: {expected_version}\n    Tatsächlich: {actual_version}",
```
Comment (dg0yt) [inline locales/messages.de.json:71]: Plural or singular (`... Port konnte nicht geladen werden, ...`)?

**Review by dg0yt — CHANGES_REQUESTED** [review]

File: locales/messages.de.json (hunk 3 above)
Comment (Neumann-A) [inline locales/messages.de.json:54]:
```suggestion
  "ProcessorArchitectureMalformed": "Fehler beim Parsen von %PROCESSOR_ARCHITECTURE% ({arch}) als eine gültige CPU-Architektur.",
```

File: locales/messages.de.json (hunk 5 above)
Comment (Neumann-A) [inline locales/messages.de.json:56]:
```suggestion
  "ProcessorArchitectureW6432Malformed": "Fehler beim Parsen von %PROCESSOR_ARCHITEW6432% ({arch}) als eine gültige CPU-Architektur. Fallback auf %PROCESSOR_ARCHITECTURE% wird durchgeführt.",
```

File: locales/messages.de.json (hunk 2 above)
Comment (Neumann-A) [inline locales/messages.de.json:65]:
```suggestion
  "VcpkgDisallowedClassicMode": "Fehler: Es wurde kein Manifest (vcpkg.json) oberhalb des aktuellen Arbeitsverzeichnisses gefunden.\nDiese vcpkg-Installation verfügt nicht über den klassischen Modus.",
```

I don't quite get how `distribution` works together with `instance` here. 
Maybe the English text needs rewording for that first.

File: locales/messages.de.json (hunk 6 above)
Comment (Neumann-A) [inline locales/messages.de.json:71]:
```suggestion
  "VersionSpecMismatch": "Fehler: Port konnte nicht geladen werden, weil die Versionsspezifikationen nicht übereinstimmen\n    Pfad: {path}\n    Erwartet: {expected_version}\n    Tatsächlich: {actual_version}",
```
Still wondering if it should be plural in the version part of the sentence.

File: locales/messages.de.json (hunk 1 above)
Comment (strega-nil-ms) [reply locales/messages.de.json:9]: I don't think it is; it's just a weird message, but that's an actual translation of the original english "Uploaded packages to {count} AWS servers"

File: locales/messages.de.json [hunk 7]
```diff
@@ -1,35 +1,35 @@
 {
   "AddPortSucceded": "Der Datei „vcpkg.json“ konnten erfolgreich Ports hinzugefügt werden.",
-  "AddTripletExpressionNotAllowed": "Fehler: Tripelausdrücke sind hier nicht zulässig. Erwägen Sie, „{package_name}:{triplet}“ in „{package_name}“ zu ändern.",
+  "AddTripletExpressionNotAllowed": "Fehler: Triplettausdrücke sind hier nicht zulässig. Erwägen Sie, „{package_name}:{triplet}“ in „{package_name}“ zu ändern.",
```
Comment (strega-nil-ms) [inline locales/messages.de.json:3]: (internal only) https://ceapex.visualstudio.com/CEINTL/_workitems/edit/565213

File: locales/messages.de.json [hunk 8]
```diff
@@ -1,35 +1,35 @@
...
   "AllFormatArgsRawArgument": "Formatzeichenfolge „{value}“ enthält ein nicht formatiertes Formatargument.",
   "AllFormatArgsUnbalancedBraces": "nicht ausgeglichene Klammer in der Formatzeichenfolge „{value}“",
   "AwsAttemptingToFetchPackages": "Es wird versucht, {count} Pakete aus AWS abzurufen.",
-  "AwsFailedToDownload": "AWS konnte nicht herunterladen, Exitcode: {exit_code}",
-  "AwsRestoredPackages": "{count} Pakete von AWS-Servern wurden in {elapsed} wiederhergestellt.",
-  "AwsUploadedPackages": "Binärdateien wurden auf {count} AWS-Server hochgeladen",
+  "AwsFailedToDownload": "Konnte nicht von AWS herunterladen, Exitcode: {exit_code}",
```
Comment (strega-nil-ms) [inline locales/messages.de.json:7]: (internal only) https://ceapex.visualstudio.com/CEINTL/_workitems/edit/565218

File: locales/messages.de.json [hunk 9]
```diff
@@ -1,35 +1,35 @@
...
+  "AwsRestoredPackages": "{count} Pakete wurden von AWS-Servern in {elapsed} wiederhergestellt.",
```
Comment (strega-nil-ms) [inline locales/messages.de.json:8]: (internal only) https://ceapex.visualstudio.com/CEINTL/_workitems/edit/565219

File: locales/messages.de.json [hunk 10]
```diff
@@ -1,35 +1,35 @@
...
+  "AwsUploadedPackages": "{count} Pakete wurden auf AWS-Server hochgeladen",
   "BothYesAndNoOptionSpecifiedError": "Fehler: --no-{option} und --{option} können nicht gleichzeitig angegeben werden.",
-  "EmptyLicenseExpression": "Der SPDX-Lizenzausdruck war leer.",
+  "EmptyLicenseExpression": "Der SPDX-Lizenzausdruck ist leer.",
```
Comment (strega-nil-ms) [inline locales/messages.de.json:11]: This is an accurate translation of the English message.

File: locales/messages.de.json [hunk 11]
```diff
@@ -1,35 +1,35 @@
...
   "ErrorIndividualPackagesUnsupported": "Fehler: Im Manifestmodus unterstützt „vcpkg install“ keine einzelnen Paketargumente.\nUm zusätzliche Pakete zu installieren, bearbeiten Sie vcpkg.json, und führen Sie dann „vcpkg install“ ohne Paketargumente aus.",
   "ErrorInvalidClassicModeOption": "Fehler: Die Option --{option} wird im klassischen Modus nicht unterstützt, und es wurde kein Manifest gefunden.",
   "ErrorInvalidManifestModeOption": "Fehler: Die Option --{option} wird im Manifestmodus nicht unterstützt.",
   "ErrorMessage": "Fehler: ",
-  "ErrorMissingVcpkgRoot": "Fehler: vcpkg-root konnte nicht erkannt werden. Wenn Sie versuchen, eine Kopie von vcpkg zu verwenden, die Sie erstellt haben, müssen Sie die VCPKG_ROOT-Umgebungsvariable definieren, um auf eine geklonte Kopie von {url} zu verweisen.",
```
Comment (strega-nil-ms) [inline locales/messages.de.json:16]: This is an accurate translation of the English message.

File: locales/messages.de.json [hunk 12]
```diff
@@ -1,35 +1,35 @@
...
-  "ErrorNoVSInstance": "Fehler: im Tripel {triplet}: Es wurde keine gültige Visual Studio-Instanz gefunden.",
+  "ErrorMissingVcpkgRoot": "Fehler: .vcpkg-root konnte nicht erkannt werden. Sollten Sie versuchen, eine selbst erstellte Kopie von vcpkg zu verwenden, muss die Umgebungsvariable VCPKG_ROOT definiert sein, um auf eine geklonte Kopie von {url} zu verweisen.",
+  "ErrorNoVSInstance": "Fehler: im Triplett {triplet}: Es wurde keine gültige Visual Studio-Instanz gefunden.",
```
Comment (strega-nil-ms) [inline locales/messages.de.json:17]: (internal only) https://ceapex.visualstudio.com/CEINTL/_workitems/edit/565228

File: locales/messages.de.json [hunk 13]
```diff
@@ -1,35 +1,35 @@
...
   "ErrorNoVSInstanceAt": "     bei „{path}“",
-  "ErrorNoVSInstanceFullVersion": "    mit dem Präfix für die Toolsetversion {version}",
-  "ErrorNoVSInstanceVersion": "    mit der Toolsetversion {version}",
+  "ErrorNoVSInstanceFullVersion": "    mit dem Präfix für die Toolset-Version {version}",
```
Comment (strega-nil-ms) [inline locales/messages.de.json:19]: (internal only) https://ceapex.visualstudio.com/CEINTL/_workitems/edit/565231

File: locales/messages.de.json [hunk 14]
```diff
@@ -1,35 +1,35 @@
...
+  "ErrorNoVSInstanceVersion": "    mit der Toolset-Version {version}",
```
Comment (strega-nil-ms) [inline locales/messages.de.json:20]: (internal only) https://ceapex.visualstudio.com/CEINTL/_workitems/edit/565234

File: locales/messages.de.json [hunk 15]
```diff
@@ -1,35 +1,35 @@
...
   "ErrorRequireBaseline": "Fehler: Diese vcpkg-Instanz erfordert ein Manifest mit einer angegebenen Baseline, um mit Ports zu interagieren. Bitte fügen Sie dem Manifest „builtin-baseline“ hinzu, oder fügen Sie eine Datei „vcpkg-configuration.json“ hinzu, welche die Standardregistrierung neu definiert.\n",
   "ErrorRequirePackagesList": "Fehler: „vcpkg install“ erfordert eine Liste von Paketen für die Installation im klassischen Modus.",
   "ErrorRequirePackagesToInstall": "Fehler: Es wurden keine Pakete für die Installation aufgelistet, und es wurde kein Manifest gefunden.",
-  "ErrorVcvarsUnsupported": "Fehler: im Tripel {triplet}: Die Verwendung der Entwickleraufforderung von Visual Studio wird auf Nicht-Windows-Hosts nicht unterstützt.\nDefinieren Sie „VCPKG_CMAKE_SYSTEM_NAME“ oder „VCPKG_CHAINLOAD_TOOLCHAIN_FILE“ in der Tripeldatei.",
+  "ErrorVcvarsUnsupported": "Fehler: im Triplett {triplet}: Die Verwendung der Entwickleraufforderung von Visual Studio wird auf Nicht-Windows-Hosts nicht unterstützt.\nDefinieren Sie „VCPKG_CMAKE_SYSTEM_NAME“ oder „VCPKG_CHAINLOAD_TOOLCHAIN_FILE“ in der Tripelettdatei.",
```
Comment (strega-nil-ms) [inline locales/messages.de.json:24]: (internal only) https://ceapex.visualstudio.com/CEINTL/_workitems/edit/565235

File: locales/messages.de.json (hunk 4 above)
Comment (strega-nil-ms) [reply locales/messages.de.json:32]: why is this pluralized?

Comment (strega-nil-ms) [reply locales/messages.de.json:32]: (internal only) https://ceapex.visualstudio.com/CEINTL/_workitems/edit/565236

File: locales/messages.de.json (hunk 3 above)
Comment (strega-nil-ms) [reply locales/messages.de.json:54]: (internal only) https://ceapex.visualstudio.com/CEINTL/_workitems/edit/565237

File: locales/messages.de.json (hunk 6 above)
Comment (autoantwort) [reply locales/messages.de.json:71]: Ingho `Paket` is a better translation, oder `Paket definition`

File: locales/messages.de.json (hunk 4 above)
Comment (autoantwort) [reply locales/messages.de.json:32]: > why is this pluralized?

There is no plural in the German sentence?

File: locales/messages.de.json [hunk 16]
```diff
@@ -51,23 +51,23 @@
...
+  "ProcessorArchitectureW6432Malformed": "Fehler beim parsen von %PROCESSOR_ARCHITEW6432% ({arch}) als eine gültige CPU-Architektur. Fallback auf %PROCESSOR_ARCHITECTURE% wird durchgeführt.",
   "SeeURL": "Weitere Informationen finden Sie unter {url}.",
   "SuggestNewVersionScheme": "Verwenden Sie das Versionsschema „{new_scheme}“ anstelle von „{old_scheme}“ in Port „{package_name}“. \nVerwenden Sie „--{option}“, um diese Überprüfung zu deaktivieren.",
-  "UnsupportedSystemName": "Fehler: VCPKG_CMAKE_SYSTEM_NAME „{system_name}“ konnte keiner vcvarsall-Plattform zugeordnet werden. Unterstützte Systemnamen sind „,“, „Windows“ und „WindowsStore“.",
-  "UnsupportedToolchain": "Fehler: Im Tripel {triplet}: Es wurde keine gültige Toolkettenkombination gefunden.\n    Die angeforderte Zielarchitektur war {arch}\n    Die ausgewählte Visual Studio-Instanz befindet sich unter {path}\n    Die verfügbaren Toolkettenkombinationen sind {list}\n",
+  "UnsupportedSystemName": "Fehler: VCPKG_CMAKE_SYSTEM_NAME „{system_name}“ konnte keiner vcvarsall-Plattform zugeordnet werden. Unterstützte Systemnamen sind „“, „Windows“ und „WindowsStore“.",
```
Comment (strega-nil-ms) [inline locales/messages.de.json:59]: (internal only) https://ceapex.visualstudio.com/CEINTL/_workitems/edit/565241

File: locales/messages.de.json (hunk 2 above)
Comment (strega-nil-ms) [reply locales/messages.de.json:65]: very likely.

Comment (strega-nil-ms) [reply locales/messages.de.json:65]: (internal only) https://ceapex.visualstudio.com/CEINTL/_workitems/edit/565252

File: locales/messages.de.json [hunk 17]
```diff
@@ -51,23 +51,23 @@
...
+  "UnsupportedToolchain": "Fehler: Im Triplett {triplet}: Es wurde keine gültige Toolset-Kombination gefunden.\n    Die angeforderte Zielarchitektur war {arch}\n    Die ausgewählte Visual Studio-Instanz befindet sich unter {path}\n    Die verfügbaren Toolset-Kombinationen sind {list}\n",
```
Comment (strega-nil-ms) [inline locales/messages.de.json:60]: (internal only) https://ceapex.visualstudio.com/CEINTL/_workitems/edit/565249

File: locales/messages.de.json (hunk 6 above)
Comment (strega-nil-ms) [reply locales/messages.de.json:71]: (internal only) https://ceapex.visualstudio.com/CEINTL/_workitems/edit/565255

File: locales/messages.de.json (hunk 4 above)
Comment (strega-nil-ms) [reply locales/messages.de.json:32]: Graphen is pluralized no?

Comment (autoantwort) [reply locales/messages.de.json:32]: Hm it's complicated. 
a graph => Ein Graph
multiple graphs => mehrere Graphen 
within a graph => in einem Graphen

**Pull request closed without merging**